### PR TITLE
mp-b2c-product-offer-prices-endpoints-tests

### PR DESCRIPTION
**Pull request by shubham-khandelwal2-metacube**
## PR Description

https://spryker.atlassian.net/browse/CC-16363
mp-b2c-product-offer-prices-endpoints-tests

### CODE REVIEW

File: tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_prices/postive.robot [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Resource    ../../../../../../resources/common/common_api.robot
+Test Setup        TestSetup
+Default Tags    glue
+
+*** Test Cases ***
+ENABLER
+    TestSetup
+
+
+Get_prices_of_a_product_offer
```
Comment (OKrivtsova) [inline tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_prices/postive.robot:12]: please add tests for other price types (you can have a look at this PR to check what prices there are ad currencies, etc.) https://github.com/spryker/robotframework-suite-tests/pull/146/files
volume prices, original price, chf currency

Comment (shubham-khandelwal2-metacube) [reply tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_prices/postive.robot:12]: DONE

File: tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_prices/postive.robot [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    And Each array element of array in response should contain nested property:    [data]    [attributes][prices][0]    currency
+    And Each array element of array in response should contain nested property with value:    [data]    [attributes][prices][0][currency][code]    ${currency_code_eur}    
+    And Each array element of array in response should contain nested property with value:    [data]    [attributes][prices][0][currency][name]    ${currency_name_eur}    
+    And Each array element of array in response should contain nested property with value:    [data]    [attributes][prices][0][currency][symbol]    ${currency_symbol_eur}    
+    And Each array element of array in response should contain nested property:    [data]    [links]    self
+    And Response body has correct self link
+    
+    # There is a bug : https://spryker.atlassian.net/browse/MP-6800
+
+    #And Each array element of array in response should contain nested property:    [data]    [attributes][prices][0][volumePrices][0]    grossAmount
```
Comment (OKrivtsova) [inline tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_prices/postive.robot:36]: if there is a bug we should not comment anything - test should fail and should stop failing when the bug is fixed
the bug reference should be moved next to the test name

File: tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_prices/postive.robot (hunk 1 above)
Comment (OKrivtsova) [reply tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_prices/postive.robot:12]: I do not see volume prices test and currencies test

File: tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_prices/postive.robot (hunk 2 above)
Comment (shubham-khandelwal2-metacube) [reply tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_prices/postive.robot]: DONE

File: tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_prices/postive.robot (hunk 1 above)
Comment (shubham-khandelwal2-metacube) [reply tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_prices/postive.robot]: DONE

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**